### ISSUE TITLE
Wildcards in script path do not work

### ISSUE THREAD

**Issue by nesnoj** [snakemake/snakemake#735]
**Snakemake version**
5.27.4

**Describe the bug**
Neither wildcard nor param can be accessed in `script:` command.

**Logs**
```
Error in rule process_layer:
    jobid: 1
    output: data/layers/municipality/output/municipality.gpkg

RuleException:
KeyError in line 50 of /home/nesnoj/git-repos/repo/Snakefile:
'wildcards'
```
**Minimal example**

I tried to use the `script:` command to execute a certain python script:
```
rule process_layer:
    output:
        process_layer_output="data/layers/{layer}/output/{layer}.gpkg"
    params:
        layer="{layer}"
    script:
        "data/layers/{wildcards.layer}/processing.py"
```
I tried to use paras instead but get the same message:
```
    script:
        "data/layers/{params.layer}/processing.py"
```

When I replace the wildcard by an explicit value e.g. `"data/layers/municipality/processing.py", everything works just fine.

**However, using the `shell:` command works perfectly (for {wildcards.layer} as well as for {params.layer})**:
```
rule process_layer:
    output:
        process_layer_output="data/layers/{layer}/output/{layer}.gpkg"
    params:
        layer="{layer}"
    shell:
        "python3 data/layers/{wildcards.layer}/processing.py"
```
Not sure whether this is a bug or simply not implemented, I could't find anything. The fact that the error message always points to the same message (see error log above) points to a general problem here.

I am grateful for any hints.. thank you!

**Comment by nesnoj** [comment]
Might be related to #732?

**Comment by nesnoj** [comment]
Fixed in #740, thanks a lot @johanneskoester !
